src/components/designSystem/Drawer.tsx

```diff
@@ -130,11 +131,16 @@ export const Drawer = forwardRef<DrawerRef, DrawerProps>(
           </Content>
 
           {!!stickyBottomBar && (
-            <StickyBottomBar className={stickyBottomBarClassName}>
+            <div
+              className={tw(
+                'border-t-grey200 sticky bottom-0 box-border border-t border-solid bg-white p-4 text-right md:px-12 md:py-4',
```

Comment:
Not sure what classes causes the issue but the sticky footer renders with a wrong border color.

Almost certain `border-t-grey200` is not a valid class but i'm not sure why the linter does not complain.

WDYT of changing all border rules with a `shadow-t` one instead? That also sets the color

Reply:
Indeed, it should have been `border-t-gray-200`, but as you've suggested, I've removed all the `border` classes and replaced them with `shadow-t`. Looks better now, good catch!

<img width="815" alt="image" src="https://github.com/user-attachments/assets/e524c1be-631d-48d6-85a3-9d98a18859a5">